README.md

```diff
@@ -1,17 +1,24 @@
-# Raven Design System
+<h1 align="center">
+   <a href="https://cuweb.github.io/rds/">
+      Raven Design System
+   </a>
+</h1>
```

Comment:
Let's remove the markup and not mess with formatting like this in a markdown file. It's ok to just have the title, it will eventually be replaced with a nice banner image